packages/desktop-client/src/components/filters/FiltersStack.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { type RuleConditionEntity } from 'loot-core/types/models/rule';
+
+import { Stack } from '../common/Stack';
+import { View } from '../common/View';
+
+import { AppliedFilters } from './AppliedFilters';
+import {
+  type SavedFilter,
+  SavedFilterMenuButton,
+} from './SavedFilterMenuButton';
+
+export function FiltersStack({
+  filters,
+  conditionsOp,
+  onUpdateFilter,
+  onDeleteFilter,
+  onClearFilters,
+  onReloadSavedFilter,
+  filterId,
+  filtersList,
+  onCondOpChange,
+}: {
+  filters: RuleConditionEntity[];
+  conditionsOp: string;
+  onUpdateFilter: (
+    filter: RuleConditionEntity,
+    newFilter: RuleConditionEntity,
+  ) => RuleConditionEntity;
+  onDeleteFilter: (filter: RuleConditionEntity) => void;
+  onClearFilters: () => void;
+  onReloadSavedFilter: () => void;
```

Comment:
Shouldn't this match the same param on savedFilterMenuButton rather than no params?